code_schemes/s01e01.json

```diff
@@ -363,6 +363,46 @@
         "StringValue": "other_theme",
         "VisibleInCoda": true
       },
+      {
+        "CodeID": "code-ee733b09",
+        "CodeType": "Normal",
+        "DisplayText": "the virus discriminates",
```

Comment:
If these are L4 codes do you want to prefix with 'rumour/misinfo' like we did for covid19-ke-urban? I'm not sure, it's up to RDA.

Reply:
I agree. We also did this in COVID19-Som. have added rumour/stigma/misinfo prefix